public/sliderdata.json

```
@@ -0,0 +1,20 @@
+{
+  "sliderdata":[
+    {
+      "id":"1" ,
+      "image":"../src/images/image-product-1.jpg"

```

Here you can't pass the full path where the image is, here you only know the name of the image you'll only have the full path if it's hosted in another service that take care of images.

so in your case the images will be in public folder, so you can remove `../src/images/` for all images and just kerp the name of the images, like so:

```suggestion
      "image":".image-product-1.jpg"
```